Week1/homework/app.js

```diff
@@ -1,11 +1,182 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+(function() {
+  const booksList = [
+    'home_and_world',
+    'the_rescue',
+    'heart_of_darkness',
+    'country_of_pointed_fires',
+    'howards_end',
+    'robinson_crusoe',
+    'the_odyssey',
+    'death_in_venice',
+    'adventures_of_huckleberry_finn',
+    'oliver_twist',
+    'frankenstein',
+    'return_of_native'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+  /*
+  function allBooks() {
+    const ul = document.createElement('ul');
+    for (let book of booksList) {
+      const li = document.createElement('li');
+      ul.appendChild(li);
+      li.setAttribute('id', book);
+      console.log(li);
+    }
+  }
+  console.log(allBooks());
+
+  */
+
+  const information = {
+    home_and_world: {
+      author: 'Rabindranath Tagore',
+      name: 'The Home and the World',
+      alt: 'This is the cover of the book called The Home and the World',
+      description:
+        "Originally written in Bengali and published in 1916, The Home and the World is the vision of Rabindranath Tagore, Asia's first Nobel laureate."
```

Comment:
Hi @YaseirAlkhwalda - As an alternative to double quotes here, you might learn about a new feature introduced in ES6 which is called template strings using back ticks i.e (`). There are also a few other benefits of using back ticks like having variables inside strings and multiline strings which you can find out here - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals